crt_portal/cts_forms/templates/forms/complaint_view/disposition/batches_table.html

```diff
@@ -41,7 +42,21 @@
                 </div>
               </td>
               {% else %}
-                <td></td>
+              <td>
+                <div class="usa-checkbox td-checkbox">
+                  <span class="usa-tooltip" title="This entire batch has been rejected and cannot be re-used">
+                    <input type="checkbox"
+                           class="usa-checkbox__input"
+                           disabled="disabled"
+                           id="checkbox-{{ datum.batch.uuid }}"
+                    >
+                    <label class="usa-checkbox__label crt-checkbox__label" for="checkbox-{{ datum.batch.uuid }}" aria-label="This entire batch has been rejected and cannot be re-used {{ datum.batch.uuid }}"></label>
```

Comment:
content nit - I think it would be clearer if we said "All reports in this batch have been rejected so a new batch cannot be generated."